modules/openapi-generator/src/main/resources/r/api.mustache

```diff
@@ -163,7 +163,7 @@
       apiResponse <- self${{{operationId}}}WithHttpInfo({{#allParams}}{{paramName}}, {{/allParams}}...)
       resp <- apiResponse$response
       if (httr::status_code(resp) >= 200 && httr::status_code(resp) <= 299) {
-        apiResponse$content
+        httr::content(resp)
```

Comment:
@LiNk-NY Do you mind explaining what issue this line aims to address?

Reply:
I've reverted this change. The internal representation should match the class rather than a simple list of contents.